src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
+    if (obj.type === 'clear') {
+      for (const keyInState in state) {
+        delete state[keyInState];
+      }
+    }
+
+    if (obj.type === 'removeProperties') {
+      for (const keyToRemove of obj.keysToRemove) {
+        for (const keyInState in state) {
+          if (keyInState === keyToRemove) {
+            delete state[keyInState];
```

Comment:
you can keyToRemove instead keyInState. If keyToRemove exists in the state, it will remove. If not - not